api/src/opentrons/protocol_api/instrument_context.py

```diff
@@ -608,7 +608,7 @@ def air_gap(
                        (Default will use all remaining volume in tip)
         :type volume: float
 
-        :param height: The number of millimiters to move above the current Well
+        :param height: The number of millimeters to move above the current Well
                        to air-gap aspirate. (Default: 5mm above current Well)
```

Comment:
```suggestion
                       to air-gap aspirate. (Default: 5 mm above current Well)
```
While we're right here. Bigger stuff can wait for sure!